docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md

```diff
@@ -2,7 +2,7 @@
 title: Introduction to Grakn
 keywords: getting started
 last_updated: September 2017
-summary: This is a brief introduction to what a knowledge based system is, what is GRAKN and the broad topics that will be covered into the Grakn Academy.
+summary: This is a brief introduction to what a knowledge graphd system is, what is GRAKN and the broad topics that will be covered into the Grakn Academy.
```

Comment:
:goat:

Reply:
Nice catch, fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f